config/routes.rb

```diff
@@ -1,7 +1,14 @@
 Rails.application.routes.draw do
+  get 'products/index'
```

Comment:
This autogenerated when creating the controller and index action, please remove this as this is handled by the resourceful route below.